subjects.json

```
@@ -679,10 +682,10 @@
     "IM21201": "OPERATIONS RESEARCH-I",
     "IM21203": "ENGINEERING ECONOMY",
     "IM22001": "INFORMATION SYSTEMS LAB",
-    "IM29202": "WORK SYSTEM DESIGN LABORATORY",
-    "IM29204": "OPERATIONS RESEARCH-II LABORATORY",
     "IM29006": "OPERATIONS RESEARCH LAB.",
     "IM29008": "WORK SYSTEM DESIGN LAB.",
+    "IM29202": "WORK SYSTEM DESIGN LABORATORY",
+    "IM29204": "OPERATIONS RESEARCH-II LABORATORY",
```

```suggestion
    "IM29202": "WORK SYSTEM DESIGN LABORATORY",
    "IM29204": "OPERATIONS RESEARCH-II LABORATORY",
```
This change distorts the sorted order of courses, please correct it.

---

Not needed